bin/php-language-server.php

```diff
@@ -10,6 +12,21 @@
     }
 }
 
-$server = new LanguageServer(new ProtocolStreamReader(STDIN), new ProtocolStreamWriter(STDOUT));
+if (count($argv) >= 3 && $argv[1] == '--tcp') {
```

Comment:
That would be nice. I can open an issue and post what I have got so far. It kind of works but there are probably better ways to do it. ;)